docker/tensorflow.dockerfile

```diff
@@ -0,0 +1,6 @@
+From pennylane/base:latest
+
+# Update and install TensorFlow
+RUN apt-get update && pip3 install tensorflow==2.5.0
```

Comment:
Running the tests after installing these packages would be a good idea for all images.